pkgs/by-name/ba/baikal/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  php,
+  fetchFromGitHub,
+  lib,
+}:
+php.buildComposerProject rec {
+  pname = "baikal";
+  version = "0.10.1";
+  src = fetchFromGitHub {
+    owner = "sabre-io";
+    repo = "Baikal";
+    tag = version;
+    hash = "sha256-YQQwTdwfHQZdUhO5HbScj/Bl8ype7TtPI3lHjvz2k04=";
+  };
+  # It doesn't provide a composer.lock file, we have to generate manually.
```

Comment:
We should probably ask upstream to include one.

Reply:
I have asked upstream, but upstream hasn't updated for 4 months.

https://github.com/sabre-io/Baikal/pull/1301